Actions/CalculateArtifactNames/CalculateArtifactNames.ps1

```diff
@@ -20,6 +18,12 @@ if ($project -eq ".") {
     $project = $settings.repoName
 }
 
+$branchName = $ENV:GITHUB_HEAD_REF
```

Comment:
Wouldn't `git branch --show-current` also do the trick? We're just trying to get the current branch we're on?

Reply:
Yes, but the current branch is something like `refs/pull/123/merge`.

Reply:
For the _head_ branch it might not matter that much, but the PR branch seems more appropriate choice.